gym_cooking/CC/parameters.txt
--num-agents 1 --level SarahDesignSingleAgent5x5 --belief-experiments --seed 1 --dqn-input Summary --single-agent  --num-eps 500000 --save-interval 10000 --folder-name singlefetcher
--num-agents 1 --level SarahDesignOne --belief-experiments --seed 1 --dqn-input Summary --single-agent  --num-eps 500000 --save-interval 10000 --folder-name singlefetcher
--num-agents 2 --level SarahDesignSingleAgent5x5 --belief-experiments --seed 1 --dqn-input Summary   --num-eps 500000 --save-interval 10000 --folder-name fetcherDQN_simpleChef
--num-agents 2 --level SarahDesignOne --belief-experiments --seed 1 --dqn-input Summary   --num-eps 500000 --save-interval 10000 --folder-name fetcherDQN_simpleChef
--num-agents 1 --level SarahDesignSingleAgent5x5 --belief-experiments --seed 2 --dqn-input Summary --single-agent  --num-eps 500000 --save-interval 10000 --folder-name singlefetcher
--num-agents 1 --level SarahDesignOne --belief-experiments --seed 2 --dqn-input Summary --single-agent  --num-eps 500000 --save-interval 10000 --folder-name singlefetcher
--num-agents 2 --level SarahDesignSingleAgent5x5 --belief-experiments --seed 2 --dqn-input Summary   --num-eps 500000 --save-interval 10000 --folder-name fetcherDQN_simpleChef
--num-agents 2 --level SarahDesignOne --belief-experiments --seed 2 --dqn-input Summary   --num-eps 500000 --save-interval 10000 --folder-name fetcherDQN_simpleChef
--num-agents 1 --level SarahDesignSingleAgent5x5 --belief-experiments --seed 3 --dqn-input Summary --single-agent  --num-eps 500000 --save-interval 10000 --folder-name singlefetcher
--num-agents 1 --level SarahDesignOne --belief-experiments --seed 3 --dqn-input Summary --single-agent  --num-eps 500000 --save-interval 10000 --folder-name singlefetcher
--num-agents 2 --level SarahDesignSingleAgent5x5 --belief-experiments --seed 3 --dqn-input Summary   --num-eps 500000 --save-interval 10000 --folder-name fetcherDQN_simpleChef
--num-agents 2 --level SarahDesignOne --belief-experiments --seed 3 --dqn-input Summary   --num-eps 500000 --save-interval 10000 --folder-name fetcherDQN_simpleChef
--num-agents 1 --level SarahDesignSingleAgent5x5 --belief-experiments --seed 4 --dqn-input Summary --single-agent  --num-eps 500000 --save-interval 10000 --folder-name singlefetcher
--num-agents 1 --level SarahDesignOne --belief-experiments --seed 4 --dqn-input Summary --single-agent  --num-eps 500000 --save-interval 10000 --folder-name singlefetcher
--num-agents 2 --level SarahDesignSingleAgent5x5 --belief-experiments --seed 4 --dqn-input Summary   --num-eps 500000 --save-interval 10000 --folder-name fetcherDQN_simpleChef
--num-agents 2 --level SarahDesignOne --belief-experiments --seed 4 --dqn-input Summary   --num-eps 500000 --save-interval 10000 --folder-name fetcherDQN_simpleChef
--num-agents 1 --level SarahDesignSingleAgent5x5 --belief-experiments --seed 5 --dqn-input Summary --single-agent  --num-eps 500000 --save-interval 10000 --folder-name singlefetcher
--num-agents 1 --level SarahDesignOne --belief-experiments --seed 5 --dqn-input Summary --single-agent  --num-eps 500000 --save-interval 10000 --folder-name singlefetcher
--num-agents 2 --level SarahDesignSingleAgent5x5 --belief-experiments --seed 5 --dqn-input Summary   --num-eps 500000 --save-interval 10000 --folder-name fetcherDQN_simpleChef
--num-agents 2 --level SarahDesignOne --belief-experiments --seed 5 --dqn-input Summary   --num-eps 500000 --save-interval 10000 --folder-name fetcherDQN_simpleChef
--num-agents 1 --level SarahDesignSingleAgent5x5 --belief-experiments --seed 6 --dqn-input Summary --single-agent  --num-eps 500000 --save-interval 10000 --folder-name singlefetcher
--num-agents 1 --level SarahDesignOne --belief-experiments --seed 6 --dqn-input Summary --single-agent  --num-eps 500000 --save-interval 10000 --folder-name singlefetcher
--num-agents 2 --level SarahDesignSingleAgent5x5 --belief-experiments --seed 6 --dqn-input Summary   --num-eps 500000 --save-interval 10000 --folder-name fetcherDQN_simpleChef
--num-agents 2 --level SarahDesignOne --belief-experiments --seed 6 --dqn-input Summary   --num-eps 500000 --save-interval 10000 --folder-name fetcherDQN_simpleChef
--num-agents 1 --level SarahDesignSingleAgent5x5 --belief-experiments --seed 7 --dqn-input Summary --single-agent  --num-eps 500000 --save-interval 10000 --folder-name singlefetcher
--num-agents 1 --level SarahDesignOne --belief-experiments --seed 7 --dqn-input Summary --single-agent  --num-eps 500000 --save-interval 10000 --folder-name singlefetcher
--num-agents 2 --level SarahDesignSingleAgent5x5 --belief-experiments --seed 7 --dqn-input Summary   --num-eps 500000 --save-interval 10000 --folder-name fetcherDQN_simpleChef
--num-agents 2 --level SarahDesignOne --belief-experiments --seed 7 --dqn-input Summary   --num-eps 500000 --save-interval 10000 --folder-name fetcherDQN_simpleChef
--num-agents 1 --level SarahDesignSingleAgent5x5 --belief-experiments --seed 8 --dqn-input Summary --single-agent  --num-eps 500000 --save-interval 10000 --folder-name singlefetcher
--num-agents 1 --level SarahDesignOne --belief-experiments --seed 8 --dqn-input Summary --single-agent  --num-eps 500000 --save-interval 10000 --folder-name singlefetcher
--num-agents 2 --level SarahDesignSingleAgent5x5 --belief-experiments --seed 8 --dqn-input Summary   --num-eps 500000 --save-interval 10000 --folder-name fetcherDQN_simpleChef
--num-agents 2 --level SarahDesignOne --belief-experiments --seed 8 --dqn-input Summary   --num-eps 500000 --save-interval 10000 --folder-name fetcherDQN_simpleChef
--num-agents 1 --level SarahDesignSingleAgent5x5 --belief-experiments --seed 9 --dqn-input Summary --single-agent  --num-eps 500000 --save-interval 10000 --folder-name singlefetcher
--num-agents 1 --level SarahDesignOne --belief-experiments --seed 9 --dqn-input Summary --single-agent  --num-eps 500000 --save-interval 10000 --folder-name singlefetcher
--num-agents 2 --level SarahDesignSingleAgent5x5 --belief-experiments --seed 9 --dqn-input Summary   --num-eps 500000 --save-interval 10000 --folder-name fetcherDQN_simpleChef
--num-agents 2 --level SarahDesignOne --belief-experiments --seed 9 --dqn-input Summary   --num-eps 500000 --save-interval 10000 --folder-name fetcherDQN_simpleChef
--num-agents 1 --level SarahDesignSingleAgent5x5 --belief-experiments --seed 10 --dqn-input Summary --single-agent  --num-eps 500000 --save-interval 10000 --folder-name singlefetcher
--num-agents 1 --level SarahDesignOne --belief-experiments --seed 10 --dqn-input Summary --single-agent  --num-eps 500000 --save-interval 10000 --folder-name singlefetcher
--num-agents 2 --level SarahDesignSingleAgent5x5 --belief-experiments --seed 10 --dqn-input Summary   --num-eps 500000 --save-interval 10000 --folder-name fetcherDQN_simpleChef
--num-agents 2 --level SarahDesignOne --belief-experiments --seed 10 --dqn-input Summary   --num-eps 500000 --save-interval 10000 --folder-name fetcherDQN_simpleChef


gym_cooking/CC/parameters_mini.txt
--num-agents 2 --level SarahDesignOne --belief-experiments --seed 1 --dqn-input Full
--num-agents 2 --level SarahDesignOne --belief-experiments --seed 1 --dqn-input Summary



gym_cooking/CC/params.sh
#!/bin/bash

# Define an array
dqn_input=("Summary")

seeds=(1 2 3 4 5 6 7 8 9 10)

single=("--single-agent" "")

eps=(500000)

intervals=(10000)

map=("SarahDesignSingleAgent5x5" "SarahDesignOne")


output_file="parameters.txt"
> "$output_file"

for item in "${dqn_input[@]}"; do
    for seed in "${seeds[@]}"; do
        for s in "${single[@]}"; do
            for ep in "${eps[@]}"; do
                for i in "${intervals[@]}"; do
                    for m in "${map[@]}"; do

                        if [[ $s == "--single-agent" ]]; then
                            echo "--num-agents 1 --level $m --belief-experiments --seed $seed --dqn-input $item $s  --num-eps $ep --save-interval $i --folder-name singlefetcher" >> "$output_file"
                        else
                            echo "--num-agents 2 --level $m --belief-experiments --seed $seed --dqn-input $item $s  --num-eps $ep --save-interval $i --folder-name fetcherDQN_simpleChef" >> "$output_file"
                        fi
                    done
                done
            done
        done
    done
done
